docs/book/src/roadmap.md

```diff
@@ -1,36 +1,21 @@
+Active community PR contributions are prioritized throughout the release and unplanned work may arise, hence the items in the milestone are a rough estimate which may change.
+The "next" milestone is where items for the milestone after the current numbered one are placed to help with a very rough bucket to help prioritize upcoming work.
 
-## v0.5 (v1alpha4) ~ Q1 2021
+## High Level Vision
 
-|Area|Description|Issue/Proposal|
-|--|--|--|
-|OS|Windows worker nodes|[#153](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/153)|
-|Identity|Multi-tenancy within one manager instance|[#586](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/586)|
-|UX|Bootstrap failure detection|[#603](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/603)|
-|UX|Add tracing and metrics|[#311](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/311)|
+CAPZ is the official production-ready Cluster API implementation to administer the entire lifecycle of self-managed or managed Kubernetes clusters (AKS) on Azure.  CAPZ enables kubernetes cluster lifecycle administration at large scale while working with GitOps and the surrounding tooling ecosystem.
 
-## v1.0 v1beta1/v1 ~ Q2 2021
+## Epics
 
-|Area|Description|Issue/Proposal|
-|--|--|--|
-|Network|Allow multiple subnets of role "node"|[#664](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/664)|
-|Network|Azure Bastion hosts|[#165](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/165)|
-|Identity|AAD Support|[#481](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/481)|
- ---
-
-## Backlog
-
-> Items within this category have been identified as potential candidates for the project
-> and can be moved up into a milestone if there is enough interest.
-
-|Area|Description|Issue/Proposal|
-|--|--|--|
-|Network|Peering of Cluster VNet to Existing VNet|[#532](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/532)|
-|OS|Flatcar Support|[#629](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/629)|
-|Compute|SGX-enabled VMs|[#488](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/488)|
-|Compute|Azure Dedicated hosts|[#675](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/675)|
-|AKS|Integrate AzureMachine with AzureManagedControlPlane|[#826](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/826)|
-|Cloud Provider|Use Out of Tree cloud-controller-manager and Storage Drivers|[#715](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/715)|
+There are a number of large priority "Epics" which may span across milestones which we believe are important to providing CAPZ users an even better experience and improving the vision.  
 
+-  Latest Core Dependencies - The foundation should keep everything under a supported version of dependencies as well as to enable the latest features. 
```

Comment:
Is the idea to keep this list a manually curated list of known release-spanning objectives? If so, should we add a note to the release documentation to audit and update this content every time we cut a minor release?

cc @CecileRobertMichon

Reply:
I don't think it should be all-inclusive of objectives spanning release milestones.  I do think it should update when one or more of this criteria happens:
1. An Epic listed has sufficiently been achieved
2. A new Epic we want to highlight surfaces
3. No updates have occurred in the past 6 months

Reply:
IMO we should have some process to regularly update the milestone that's independent of the release cycle (or maybe every few releases?). It's a good idea to embed it into some other existing process (eg. release) though as it helps ensure we don't forget about it and let it get stale as we have.

Reply:
Agree, we should state that updating this doc as part of the release cycle is conditional and that it is normal for this high level set of objectives to remain static over a few release cycles sometimes.

I wish there were a part of the release documentation where we say "these are the things you do a week before you plan to cut a release" so that we can have ample time to do these more manual, holistic tasks in advance of the actual release.

Reply:
> I wish there were a part of the release documentation where we say "these are the things you do a week before you plan to cut a release" so that we can have ample time to do these more manual, holistic tasks in advance of the actual release.

like this https://github.com/kubernetes-sigs/cluster-api/issues/new?assignees=&labels=&template=release_tracking.md&title=Tasks+for+v%3Crelease-tag%3E+release+cycle? We could add one...

Reply:
What if we just say every even or odd release number we put a note to check/update the roadmap doc?

Reply:
that seems a bit arbitrary... Maybe every release we check it but we don't always update it

Reply:
Ok, sure.

Reply:
@dtzar want to add that step to https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/docs/book/src/developers/releasing.md?